Specs/DateTime/Spanish/DatePeriodExtractor.json

```diff
@@ -463,67 +378,27 @@
   },
   {
     "Input": "Estare afuera el tercer cuatrimestre de 2016",
+    "NotSupported": "dotnet",
+    "NotSupportedByDesign": "javascript, python, java",
     "Results": [
       {
-        "Text": "el tercer cuatrimestre de 2016",
-        "Type": "daterange",
-        "Start": 14,
-        "Length": 30
-      }
-    ]
-  },
-  {
-    "Input": "Estare afuera el tercer cuatrimestre de este año",
-    "Results": [
-      {
-        "Text": "el tercer cuatrimestre de este año",
```

Comment:
In English, this test resolves `the third quarter this year` from the input `I'll be out the third quarter this year` maintaining `the` keyword, so in Spanish, we should keep `el` equivalent.